src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss

```diff
@@ -28,82 +32,76 @@
-            }
-        }
-
-        .bcs-version-item + .bcs-activity-feed-annotation-activity {
-            margin-top: $bdl-grid-unit * 4;
-        }
-
-        .bcs-activity-feed-task,
-        .bcs-activity-feed-task-new,
-        .bcs-activity-feed-comment,
-        .bcs-activity-feed-app-activity,
-        .bcs-activity-feed-annotation-activity {
+        .bcs-ActivityItem {
             position: relative;
-            padding-top: 20px;
-            padding-bottom: 20px;
-            border-bottom: 1px solid $bdl-gray-10;
-
-            &:first-child .bcs-comment {
-                margin-top: 0;
-            }
+            padding: 20px $container-padding;
+            box-shadow: 0 $container-padding 0 -#{$container-padding - 1} $bdl-gray-10;
+            backface-visibility: hidden; // Prevent visual shifting during pseudo-element scale transitions
+            transition: box-shadow $container-transition-duration ease $container-transition-delay;
 
             &.bcs-is-focused {
-                $container-padding: 25px;
-
-                margin-right: -$container-padding;
-                margin-bottom: 1px;
-                margin-left: -$container-padding;
-                padding-right: $container-padding;
-                padding-left: $container-padding;
-                background-color: $bdl-box-blue-05;
-                border-bottom: none;
-                box-shadow: 0 $container-padding 0 -#{$container-padding - 1} $bdl-gray-10;
-            }
-        }
-
-        .bcs-activity-feed-task,
-        .bcs-activity-feed-task-new,
-        .bcs-activity-feed-comment,
-        .bcs-activity-feed-app-activity {
-            &.bcs-is-focused {
-                animation: fade-out 6s ease-out 2s 1 normal forwards;
+                box-shadow: 0 0 $bdl-grid-unit * 4 $bdl-gray-30;
+                animation: fade-out 1s ease-in-out $container-transition-delay 0 normal none; // Disabled by default
 
                 @keyframes fade-out {
-                    0% {
-                        background-color: $bdl-box-blue-05;
-                    }
-
+                    0%,
                     100% {
                         background-color: transparent;
                     }
+
+                    25%,
+                    75% {
+                        background-color: $bdl-box-blue-05;
+                    }
+                }
+            }
+
+            &:last-child {
+                margin-bottom: 10px;
+            }
+
+            &:not(:focus-within) {
+                &.bcs-is-focused {
+                    animation-iteration-count: 1;
+                    animation-fill-mode: forwards;
                 }
             }
         }
 
-        .bcs-activity-feed-annotation-activity.bcs-is-focused {
-            padding-left: 21px;
-            border-left: 4px solid $bdl-yellorange;
+        .bcs-version-item {
+            margin-top: $bdl-grid-unit * 3;
+            margin-bottom: $bdl-grid-unit * 3;
```

Comment:
Did we want to maintain or come up with a new rule to apply a margin-top to the `. bcs-activity-feed-annotation-activity` when it follows a `.bcs-version-item`?